### PR TITLE
add sentio to tooling documentations

### PR DESCRIPTION
**Pull request by qiaokan**
Hi,
  This is Kan from sentio.xyz. adding sentio to the tooling documentation.

### CODE REVIEW

File: docs/build-on-linea/tooling/data-indexers/sentio.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+sidebar_position: 3
+---
+
+# Sentio
+
+Sentio provides modern **analytics**, **monitoring**, **alerting and debugging** for decentralized applications as an **integrated** and **hosted** experience.
+
+Users can use Sentio’s SDK to **collect** and **transform** [Metrics](https://docs.sentio.xyz/references/concepts/data-types/metrics) and [event-logs](https://docs.sentio.xyz/references/concepts/data-types/event-logs) generated based on their smart contracts’ events, transactions, traces and states. Sentio fully manages the indexing process, and users can visualize data in the [dashboard](https://docs.sentio.xyz/how-to-guides-by-examples/visualize-results/build-dashboards), set up [alerts](https://docs.sentio.xyz/references/concepts/alerts), perform advanced search on the logs, and dig into a transaction via [Sentio debugger](https://docs.sentio.xyz/sentio-debugger), all via Sentio’s web interface.
+
+<figure><img src="https://github.com/sentioxyz/docs/blob/main/.gitbook/assets/all.gif" alt=""></figure>
```
Comment (emilyJLin95) [inline docs/build-on-linea/tooling/data-indexers/sentio.md:12]: there's no closing tag for `img`

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]
hihi!

Can you delete the `yarn.lock` file - is it necessary?

Also, your build is failing I think because of a missing tag
Is it building locally for you? you can find instructions here: https://docs-template.consensys.net/contribute/preview

**Comment by qiaokan** [comment]
> hihi!
> 
> Can you delete the `yarn.lock` file - is it necessary?
> 
> Also, your build is failing I think because of a missing tag Is it building locally for you? you can find instructions here: https://docs-template.consensys.net/contribute/preview

Thanks for the review!
yes. removed yarn.lock.
i am able to build/run it locally.

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**